queries.sql

```diff
@@ -1,3 +1,56 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name FROM animals WHERE (escape_attempts < 3) AND neutered;
+SELECT date_of_birth from animals WHERE name IN ('Agumon', 'Pikachu');
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered;
+SELECT * FROM animals WHERE name NOT IN ('Gabumon');
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
```

Comment:
- [ ] It would be better if we verify the change was undone after the rollback
![Screenshot from 2023-08-14 06-07-12](https://github.com/aamir-asaram/vet-clinic/assets/46388113/64d591d4-8fd0-4366-8410-1a010d6dcb28)
As seen in the highlighted text